### DISCUSSION TITLE
[WASM AOT] Setting Image.Source not working?!?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#10605] [Q&A] [answered]
I see this error in browser console:
`UnusualSuite.png:1          GET https://stage.unusual-suite.com/package_39e2eda6c76d240c639ef9b04d315d042df3080a/Assets/UnusualSuite.png 404`

I have code like this in place:
`new Image { Source = new BitmapImage { UriSource = new Uri($"ms-appx:///Assets/UnusualSuite.png") } }`

'UnusualSuite.png' is available in the shared project with 'Build action = Content'.

What am I doing wrong?

**Comment by jeromelaban** [comment]
Thanks for the report. You server may report 404 if the casing is not exactly the same as the one specified in the Uri, depending on where it is hosted.

Still, this is something Uno could reduce friction for by using the known casing for your app. Let's track in https://github.com/unoplatform/uno/issues/10606.

**Reply by DierkDroth** [reply]
@jeromelaban thanks for your feedback.

I just checked again: the casing of the actual PNG file name in my project does match the casing of the resource string in code snippet above.

What am I missing?

**Reply by jeromelaban** [reply]
I do not know. If the file is not served, it means that the file is blocked from being served. Make sure to check that your server has the right configuration.

**Reply by DierkDroth** [reply]
@jeromelaban thanks again. 

I will consult with my web admin.

**Comment by DierkDroth** [comment, marked as answer]
There are several issues intertwined. Closing the case in favor of https://github.com/unoplatform/uno/discussions/10616 which I was able to isolate.
